### PR TITLE
plumb reset info through to KBFS

### PR DESCRIPTION
**Pull request by maxtaco**
- via `resets` field in `UserPlusKeys`

cc: @strib, is this what you guys need?

### CODE REVIEW

File: protocol/avdl/keybase1/reset.avdl [hunk 1]
```diff
@@ -45,4 +45,10 @@ protocol Reset {
     ResetType type;
   }
 
+  // ResetSummaryWithEldestSeqno also includes the EldestSeqno at the time of the reset
```
Comment (mlsteele) [inline protocol/avdl/keybase1/reset.avdl]: Before the reset?

Comment (maxtaco) [reply protocol/avdl/keybase1/reset.avdl]: Yeah, right before the reset...

**Comment by maxtaco** [comment]
I can add it to reset summary, good point.

**Comment by maxtaco** [comment]
OK PTAL!

File: protocol/avdl/keybase1/reset.avdl [hunk 2]
```diff
@@ -23,7 +23,7 @@ protocol Reset {
     @jsonkey("eldest_kid")
     union { null, KID } eldestKID;
     @jsonkey("public_seqno")
-    Seqno eldestSeqno;
+    Seqno lastSeqno;
```
Comment (maxtaco) [inline protocol/avdl/keybase1/reset.avdl]: this was named improperly

**Pull request merged**